models/aeif_cond_alpha.h

```diff
@@ -121,16 +140,17 @@ The following parameters can be set in the status dictionary.
  w       pA      Spike-adaptation current
 ======== ======= =======================================
 
-======== ======= =======================================
+=========== ======= =======================================
 **Membrane Parameters**
 --------------------------------------------------------
```

Comment:
The Membrane Parameter tables (both models) do not render due to:
"Column span incomplete in table line 3."

```suggestion
-----------------------------------------------------------
``